mir-module/src/main/resources/xsl/editor/mir2xeditor.xsl

```diff
@@ -204,6 +237,13 @@
             <xsl:value-of select="i18n:translate('mir.date.period')" />
           </a>
         </li>
+        <xsl:if test="$showDateTimeOption='true'">
```

Comment:
... but compared to a string literal.

I think, the default values should be string literals as well.